src/api-service/__app__/onefuzzlib/extension.py

```diff
@@ -94,9 +84,24 @@ def dependency_extension(region: Region, vm_os: OS) -> Optional[Extension]:
         return None
 
 
-def build_pool_config(pool_name: str) -> str:
+def build_scaleset_script(pool: Pool, scaleset: Scaleset) -> str:
+    commands = []
+    extension = "ps1" if pool.os == OS.windows else "sh"
+    filename = f"{scaleset.scaleset_id}/scaleset-setup.{extension}"
+    sep = "\r\n" if pool.os == OS.windows else "\n"
+
+    if pool.os == OS.windows and scaleset.auth is not None:
+        ssh_key = scaleset.auth.public_key.strip()
+        ssh_path = "$env:ProgramData/ssh/administrators_authorized_keys"
+        commands += ['Set-Content -Path %s -Value "%s"' % (ssh_path, ssh_key)]
```

Comment:
Is there a reason we don't use f-strings here, though we do above?

Reply:
Habit.  Updated.